packages/manager/modules/vps/src/header/vps-header.controller.js

```diff
@@ -83,7 +83,7 @@ export default class {
 
   checkMessages(vps) {
     this.isExpired(vps);
-    this.isInRescueMode(vps.netbootMode);
+    this.warnForRescueMode(this.isInRescueMode);
```

Comment:
`warn` ? What does this do ? It shows a dialog or a message ?

Reply:
It calls `CucCloudMessage.warning` to warn user that VPS is in rescue mode.
(https://github.com/ovh/manager/pull/2853/files#diff-fb48e0095e7b9f1ceac12ef1908e6342R106)

Reply:
`displayWarning` instead maybe ?